migrations/20161206123726-gotbook.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+module.exports = {
+  up: function (queryInterface, Sequelize) {
+    return queryInterface.createTable('Gotbooks', {
```

Comment:
Gotbooks, bad database table name. It should be ReaderBookRequests table based on what I read below